src/olm/messages/message.rs

```diff
@@ -264,6 +389,49 @@ struct ProtoBufMessage {
     ciphertext: Vec<u8>,
 }
 
+#[cfg(feature = "interolm")]
+#[derive(PartialEq, Eq, ProstMessage)]
+struct InterolmProtoBufMessage {
```

Comment:
Seems a bit odd to split `ProtoBufMessage` from its implementation, did you mean for this bit to go below `impl ProtoBufMessage`?

Reply:
Switched it around, thanks!